docs/changelog.rst

```diff
@@ -173,6 +173,8 @@ New features:
 * :doc:`/plugins/replaygain` now does its analysis in parallel when using
   the ``command``, ``ffmpeg`` or ``bs1770gain`` backends.
   :bug:`3478`
+* The ``mbsync`` plugin now also fetches performers, if the ``performer_info``
+  option is enabled. Fixes :bug`1547`, thanks to :user:`dosoe`.
```

Comment:
```suggestion
  option is enabled. Fixes :bug:`1547`, thanks to :user:`dosoe`.
```

Reply:
thanks for your vigilance!

Reply:
Sorry I didn't merge your suggestion, I'm still struggling with git.